test/integration/submission/submission.spec.ts

```diff
@@ -589,10 +589,44 @@ describe('Submission Api', () => {
         .auth(JWT_ABCDEF, { type: 'bearer' })
         .end(async (err: any, res: any) => {
           try {
-            res.should.have.status(400);
-            res.body.should.deep.eq({
-              msg: `invalid file name, must start with ${ClinicalEntityType.REGISTRATION} and have .tsv extension`,
+            res.should.have.status(422);
+            res.body.batchErrors.should.deep.include({
+              message: `Please retain the template file name and only append characters to the end. For example, sample_registration<_optional_extension>.tsv`,
```

Comment:
can you put the messages in constants at the top of the test, it will be easier to maintain in case of change